src/Itinerary/Contexts/ICT001Itineraries/Itineraries.tsx

```diff
@@ -1,60 +1,68 @@
+import { ActivityIndicator } from 'react-native-paper';
 import { useState, createContext, useEffect, ReactNode } from 'react';
 import db from '@/Itinerary/Endpoint/firestore'
-
-import { collection, onSnapshot, doc, addDoc, getDoc, DocumentReference, DocumentSnapshot } from 'firebase/firestore';
-
+import { collection, doc, query, QuerySnapshot, onSnapshot, addDoc, DocumentReference, DocumentSnapshot, QueryDocumentSnapshot } from 'firebase/firestore';
+import { IDB001ItinerariesCols, collectionName, IDB001ItinerariesInterface } from '@/Itinerary/Models/IDB001Itineraries'
 import { ICT001ItinerariesInterface } from './ItinerariesInterface';
 import { ICT001ItinerariesConverter } from './ItinerariesConverter';
+import { ICT001ItinerariesBuild } from './ItinerariesBuild';
 
+/**
+ * Define value interface of useContext(ICT001Itineraries). 
+ */
 interface ICT001ItinerariesValInterface {
     documentSnapshot: DocumentSnapshot<ICT001ItinerariesInterface>;
-    create: () => Promise<DocumentReference<ICT001ItinerariesInterface>>;
+    create: () => void;
```

Comment:
```suggestion
    create: () => Promise<void>;
```
@Ayato-kosaka 
挙動関係ないけど、
後に読み返したときに関数の中身見なくても、
戻り値がPromiseって書いてあれば、非同期関数なんだなってわかりやすいかな

Reply:
おけ〜[9cf59e9](https://github.com/Ayato-kosaka/spelieve/pull/137/commits/9cf59e98ab9347695d04794e491d202a514ee5b6)で対応した